core/src/BibleEngine.class.ts

```diff
@@ -777,16 +780,21 @@ export class BibleEngine {
                     versionId: state.book.versionId,
                     phraseNum: state.currentPhraseNum
                 };
+
                 const phraseId = generatePhraseId(phraseRef);
+                if (phraseId in state.alreadyUsedPhraseIds) {
+                    throw new Error('Duplicate phrase id found.');
+                }
+                state.alreadyUsedPhraseIds[phraseId] = content;
```

Comment:
see comment above.

apart from that, it isn't necessary to add content to that data structure or? the purpose is only to keep track of all used phraseIds i suppose?